Testsuite/test/parse-ctest-dashboard-xml.py

```diff
@@ -71,9 +71,10 @@ def open_file_create_dir(filename, mode_, *args, **kwargs):
 for t_id in range(0, len(tests)):
     t = tests[t_id]
     for l in t['Labels']:
-        label = l.replace("_Tests","")
-        labels.add(label)
-        tests_per_label[label].append(t)
+        if "_Tests" in l or "_Examples" in l or "_Demo" in l:
+          label = l.replace("_Tests","")
+          labels.add(label)
+          tests_per_label[label].append(t)
```

Comment:
The indentation seems to be 2 spaces, whereas it is 4 spaces in the other lines. Does not it cause issues, with Python?

Reply:
it should, you are right.